.github/workflows/issue-opened.yml

```
@@ -0,0 +1,28 @@
+name: Run commands when issues are opened
+on:
+  issues:
+    types: [opened]
+concurrency:
+  group: issue-commands-${{ github.event.issue.number }}
```

```suggestion
  group: issue-opened-${{ github.event.issue.number }}
```

this code prevents racing condition and parallel running of the same action per issue, it has to have a unique name otherwise you could have this action run being cancelled out by a `labeled` one or other way around.